### ISSUE TITLE
Add without reparent

### ISSUE THREAD

**Issue by Styx2592** [rlewicki/AdvancedSightSupport#5]
Greetings!
I has asked you in the unreal questions before about to implement the interface at your own, because i already has a parent for my character thats also a c++ class.
You answered me, but you told me a way but i dont has a cpp project and i also cant programm c++. So i need a way to implement the interface in a blueprint only project.
My character parent class is also needed because i use an other c++ asset for my characters. But i dont have any experience with that.

Is that possible? Than i would buy your system direclty :D

**Comment by rlewicki** [comment]
Hi @Styx2592 thanks for reaching out! Before I can help you the main question is whether you are able to compile C++ code on your own? I'm asking because even though I can guide you how to update the code, in the end you will need to build your own binaries. This means you need to have a Visual Studio installed on your PC etc.

Let me know if this is something you can do, and if not whether you are willing to do so, I can then help you figure it out.

**Comment by Styx2592** [comment]
No, i dont have experience with compliling and i dont have visual studio etc on my computer.

Is that the only way to implement your code?

**Comment by rlewicki** [comment]
Unfortunately this is how this works. Just updating the code isn't going to change anything in the editor because the code after a change is made needs to be compiled. This is why plugins from marketplace are distributed with binaries (they are inside plugin's folder, inside `Binaries` directory).

This means that I can help you with writing the code, but since you also depend on another plugin that I don't have access to, this means I cannot help you with providing binaries.

The only solution is to avoid editing code at all or compiling your code changes.

Now that I think about it I might also implement my own interface that would allow doing so in blueprints instead requiring C++ compilation but that would take me few days to do so, and I will also have to be cautious to ensure I don't break any existing logic for other users.

**Comment by Styx2592** [comment]
That would be great! I can definitely wait a few days. 
I think there are many assets like locomotion or combat frameworks that run on cpp and therefore many potential customers get similar problems. That would certainly help your product and I could definitely use it. 

Would you make these customizations?

**Comment by rlewicki** [comment]
Yes, I will try and implement in this week or next week latest. Will update you as soon as it's available.

**Comment by rlewicki** [comment]
Hi @Styx2592! I'm happy to announce that new version (v1.2) has been published to the marketplace and can now be downloaded. Changelog for the new version is available in the plugin's description on the marketplace page. Cheers!
